docker/l4t/32.1/jax-tx2/Dockerfile

```diff
@@ -1,10 +1,22 @@
 ARG VERSION_ID
+# using a smaller image doesn't save us any space as the layers are used by the other images.
+FROM ubuntu:${VERSION_ID} as qemu
+
+# install qemu for the support of building containers on host
+RUN apt-get update && apt-get install -y --no-install-recommends qemu-user-static && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# start of the real image base
 FROM arm64v8/ubuntu:${VERSION_ID}
 
 ARG URL=https://developer.download.nvidia.com/embedded/L4T/r32_Release_v1.0/jax-tx2/BSP
 
 ARG DRIVER_PACK=JAX-TX2-Jetson_Linux_R32.1.0_aarch64.tbz2
 
+# pre-req qemu binary for cross-compilation on x86
```

Comment:
# pre-req QEMU binary for aarch64 emulation on x86_64

This isn't cross-compilation and I think we should leave cross-compilation comments for that container.

Reply:
Sure. I will update this.